pontoon/base/views.py

```diff
@@ -603,6 +604,24 @@ def add_comment(request):
     return JsonResponse({"status": True})
 
 
+@login_required(redirect_field_name="", login_url="/403")
+@require_POST
+@transaction.atomic
+def update_comment(request):
+    """ Update pinned status of comment """
```

Comment:
I chose to keep the naming neutral (rather than something like update_pinned_comment) in case we ever want to add additional update actions, but then added the note to indicate that only the pinned status is being updated ATM.